supporter-product-data/cloudformation/cfn.yaml

```diff
@@ -154,20 +168,39 @@ Resources:
         Variables:
           'Stage': !Sub ${Stage}
 
-  UpdateDynamoLambda:
+  AddSupporterRatePlanItemToQueueLambda:
     Type: "AWS::Lambda::Function"
     Properties:
       FunctionName:
-        !Sub support-SupporterProductDataUpdateDynamo-${Stage}
+        !Sub support-SupporterProductDataAddSupporterRatePlanItemToQueue-${Stage}
       Description: "Update DynamoDB with the results of the Zuora query"
-      Handler: "com.gu.lambdas.UpdateDynamoLambda::handleRequest"
+      Handler: "com.gu.lambdas.AddSupporterRatePlanItemToQueueLambda::handleRequest"
+      Role: !GetAtt [ SupporterProductDataLambdaRole, Arn ]
+      Code:
+        S3Bucket: supporter-product-data-dist
+        S3Key: !Sub support/${Stage}/supporter-product-data/supporter-product-data.jar
+      MemorySize: 4096
+      Runtime: "java8.al2"
+      Timeout: 600
+      Environment:
+        Variables:
+          'Stage': !Sub ${Stage}
+
+  ProcessSupporterRatePlanItemLambda:
+    Type: "AWS::Lambda::Function"
+    Properties:
+      FunctionName:
+        !Sub support-SupporterProductDataProcessSupporterRatePlanItem-${Stage}
+      Description: "Process new subscriptions from the SQS queue"
+      Handler: "com.gu.lambdas.ProcessSupporterRatePlanItemLambda::handleRequest"
       Role: !GetAtt [ SupporterProductDataLambdaRole, Arn ]
       Code:
         S3Bucket: supporter-product-data-dist
         S3Key: !Sub support/${Stage}/supporter-product-data/supporter-product-data.jar
       MemorySize: 4096
       Runtime: "java8.al2"
```

Comment:
consider using java11 now - it's the recommended one for new stuff, and it should be more efficient too.